Geometry/TrackerNumberingBuilder/plugins/TrackerGeometricDetESModule.cc

```diff
@@ -24,9 +24,30 @@ using namespace edm;
 
 TrackerGeometricDetESModule::TrackerGeometricDetESModule( const edm::ParameterSet & p ) 
   : fromDDD_( p.getParameter<bool>( "fromDDD" )),
-    layerNumberPXB_( p.exists( "layerNumberPXB" ) ? p.getParameter<unsigned int>( "layerNumberPXB" ) : 16U ),// 16 for current, 18 for SLHC 
-    totalBlade_( p.exists( "totalBlade" ) ? p.getParameter<unsigned int>( "totalBlade" ) : 24U )             // 24 for current, 56 for SLHC 
```

Comment:
by "break the HLT" - what test do you mean exactly? We certainly do not have a mass failure in the IBs.

On Jan 6, 2015, at 8:04 AM, Martin Grunewald <notifications@github.com>
 wrote:

> In Geometry/TrackerNumberingBuilder/plugins/TrackerGeometricDetESModule.cc:
> 
> > @@ -24,9 +24,30 @@ using namespace edm;
> >  
> >  TrackerGeometricDetESModule::TrackerGeometricDetESModule( const edm::ParameterSet & p ) 
> >    : fromDDD_( p.getParameter<bool>( "fromDDD" )),
> > -    layerNumberPXB_( p.exists( "layerNumberPXB" ) ? p.getParameter<unsigned int>( "layerNumberPXB" ) : 16U ),// 16 for current, 18 for SLHC 
> > -    totalBlade_( p.exists( "totalBlade" ) ? p.getParameter<unsigned int>( "totalBlade" ) : 24U )             // 24 for current, 56 for SLHC 
> 
> Because it is not tested --- sigh! I make a PR....
> 
> —
> Reply to this email directly or view it on GitHub.
>